### ISSUE TITLE
Brain fog

### ISSUE THREAD

**Issue by ahamosh** [obophenotype/human-phenotype-ontology#6661]
**Preferred term label:**

**Synonyms**
Fuzzy brain, Fuzzy headed,

**Definition (free text, please give PubMed ID)**
An inability to think clearly.

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
Adding this term with the following definition

Def: Brain fog is a type of transient cognitive dysfunction that comprises a constellation of symptoms that impair intellectual functioning to a level that interferes with daily activities, commonly including forgetfulness, mental slowness, difficulty thinking or focusing, inability to find the right words, a sensation that the mind went blank or is "cloudy". Brain fog tends to recur and may be triggered by factors such as physical fatigue, lack of sleep, and prolonged standing or may appear to occur spontaneously.


Comment:
Brain fog has been reported in a range of diseases including myalgic encephalomyelitis/chronic fatigue syndrome
non-celiac gluten disease,postural tachycardia syndrome, lupus, and COVID-19.
PMID:23999934 
PMID:32857796
PMID:33199820 
PMID:30452327
PMID:32984564

Synonyms Mental fog, Mental clouding, or Mental fatigue.

**Comment by pnrobinson** [comment]
added
